financialaid/models.py

```diff
@@ -0,0 +1,110 @@
+"""
+Models for the Financial Aid App
+"""
+
+from django.contrib.auth.models import User
+from django.db import (
+    models,
+    transaction,
+)
+from jsonfield import JSONField
+
+from courses.models import (
+    CourseRun,
+    Program,
+)
+
+
+class Tiers(models.Model):
+    """
+    The possible tiers to be used
+    """
+    name = models.TextField()
+    description = models.TextField()
+
+
+class TierProgram(models.Model):
+    """
+    The tiers for discounted pricing assigned to a program
+    """
+    program = models.ForeignKey(Program, null=False)
+    tier = models.ForeignKey(Tiers, null=False)
+    value = models.IntegerField(null=False)
+    current = models.BooleanField(null=False, default=False)
+
+    class Meta:
+        unique_together = ('program', 'tier')
+
+    @transaction.atomic
+    def save(self, *args, **kwargs):
+        """
+        Override the save to enforce the existence of only one `current` = True
+        per program and tier
+        """
+        if self.current:
+            TierProgram.objects.filter(program=self.program, tier=self.tier, current=True).update(current=False)
+        return super(TierProgram, self).save(*args, **kwargs)
+
+
+class FinalcialAidStatus:
+    """Statuses for the Financial Aid model"""
+    CREATED = 'created'
```

Comment:
my idea is that when the entry is created the first time it gets `created`, then it changes to `auto_approved` or `pending_docs ` according to the algorithm.
This is mainly because I want the `FinancialAidAudit` to have all the log of the changes and the timestamps.